hz: 10          # ループ周期 [Hz]
map_size: 4.0   # マップの一辺の長さ [m]
map_reso: 0.025 # マップの解像度 [m/cell]
